app/assets/src/components/views/SampleViewV2/ReportTableV2.jsx

```diff
@@ -0,0 +1,404 @@
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).contig_reads || 0,
+            (rowData.nr || {}).contig_reads || 0,
+          ];
+        },
+        cellRenderer: this.renderNtNrDecimalValues,
+        dataKey: "contig_reads",
+        label: "contig r",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["contig_reads"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 75,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).percent_identity || 0,
+            (rowData.nr || {}).percent_identity || 0,
+          ];
+        },
+        cellRenderer: ({ cellData }) =>
+          this.renderNtNrDecimalValues({ cellData, decimalPlaces: 1 }),
+        dataKey: "percent_identity",
+        label: "%id",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["percent_identity"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 60,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).alignment_length || 0,
+            (rowData.nr || {}).alignment_length || 0,
+          ];
+        },
+        cellRenderer: ({ cellData }) =>
+          this.renderNtNrDecimalValues({ cellData, decimalPlaces: 1 }),
+        dataKey: "alignment_length",
+        label: "L",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["alignment_length"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 60,
+      },
+      {
+        cellDataGetter: ({ rowData }) => {
+          return [
+            (rowData.nt || {}).e_value || 0,
+            (rowData.nr || {}).e_value || 0,
+          ];
+        },
+        cellRenderer: ({ cellData }) =>
+          this.renderNtNrDecimalValues({ cellData, decimalPlaces: 1 }),
+        dataKey: "e_value",
+        label: "log(1/E)",
+        sortFunction: ({ data, sortDirection }) =>
+          this.nestedNtNrSortFunction({
+            data,
+            sortDirection,
+            path: ["e_value"],
+            nullValue: 0,
+            limits: NUMBER_NULL_VALUES,
+          }),
+        width: 60,
+      },
+      {
+        dataKey: "ntnrSelector",
+        disableSort: true,
+        headerClassName: cs.ntnrSelectorHeader,
+        headerRenderer: this.renderNtNrSelector,
+        width: 40,
+      },
+    ];
+  }
+
+  nestedSortFunction = ({ data, path, sortDirection, nullValue, limits }) => {
```

Comment:
This seems to be the most complex function in this class. Could you add some more comments?

Reply:
Added some.